pwiz_tools/Skyline/FileUI/PeptideSearch/ImportPeptideSearchDlg.cs

```diff
@@ -544,7 +544,17 @@ private void NextPage()
                             CloseWizard(DialogResult.Cancel);
                         }
 
-                        if (ImportResultsControl.ResultsFilesMissing)
+                        var anyResults = ImportResultsControl.FoundResultsFiles.Any();
+                        if (!anyResults)
+                        {
+                            if (MessageBox.Show(this,

```

Comment:
Also, these need to be changed to MultiButtonMessageDlg() to support writing tests for this case. We should get this added to Brian Pratt's static code analysis tool. We don't really want any use of MessageBox in the Skyline code, because it can't be dismissed in an automated test, which kind of proves the other message is not covered in an automated test.